From abb53bd7f51366b4bb62ba2f9fb987e8f7717937 Mon Sep 17 00:00:00 2001
From: Bin Yang <yangbin@rock-chips.com>
Date: Wed, 20 Nov 2019 11:03:32 +0800
Subject: [PATCH] mfd/fusb302: Set CC pull up current to 80uA

The current driver set CC pull up current to 180uA, but FUSB302B was
designed with 80uA for sink detection which means it only guarantee
FUSB302B will works well with that setting. So we change the CC pull
up current to 80uA.

Change-Id: I5bcb0caaffafbcdf7972396b64f296606bbf3986
Signed-off-by: Bin Yang <yangbin@rock-chips.com>
---
 drivers/mfd/fusb302.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mfd/fusb302.c b/drivers/mfd/fusb302.c
index a120e5764ecc..a7d460ae224d 100644
--- a/drivers/mfd/fusb302.c
+++ b/drivers/mfd/fusb302.c
@@ -774,7 +774,7 @@ static void tcpm_init(struct fusb30x_chip *chip)
 	val = ~MASKB_M_GCRCSEND;
 	regmap_write(chip->regmap, FUSB_REG_MASKB, val);
 
-	tcpm_select_rp_value(chip, TYPEC_RP_1A5);
+	tcpm_select_rp_value(chip, TYPEC_RP_USB);
 	/* Interrupts Enable */
 	regmap_update_bits(chip->regmap, FUSB_REG_CONTROL0, CONTROL0_INT_MASK,
 			   ~CONTROL0_INT_MASK);
-- 
2.35.3

